src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding-top: 32px;
+  padding-left: 16px;
+  padding-right: 16px;
+  align-items: center;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)